docs/conf.py

```diff
@@ -141,6 +141,8 @@
         "version_match": tag_or_branch,
     },
     "navbar_start": ["navbar-logo", "version-switcher"],
+    "navbar_persistent": [],
```

Comment:
This line helps with removing the persistent search button. It was not available in v11.0 so upgraded the theme to v12.0. Now it should be consistent with what we had on the website before.